ee/hogai/graph/base.py

```diff
@@ -102,5 +127,19 @@ def _message_to_langgraph_update(
         """
         return ((), "messages", (message, {"langgraph_node": node_name}))
 
+    async def _write_message(self, message: AssistantMessageUnion, node_name: MaxNodeName | None = None):
+        """
+        Writes a message to the stream writer.
+        """
+        writer = await self._get_writer()
+        if writer and self.node_name:
+            writer(self._message_to_langgraph_update(message, self.node_name))
+
+    async def _stream_reasoning(self, content: str, substeps: list[str] | None = None):
```

Comment:
We don't stream reasoning but just write it to the channel:
```suggestion
    async def _write_reasoning(self, content: str, substeps: list[str] | None = None):
```